models/classes/runner/config/QtiRunnerConfig.php

```diff
@@ -55,42 +55,42 @@ protected function buildConfig() {
-                'useTitle' => !empty($rawConfig['test-taker-review-use-title']),
-                'forceTitle' => !empty($rawConfig['test-taker-review-force-title']),
-                'showLegend' => !empty($rawConfig['test-taker-review-show-legend']),
-                'defaultOpen' => !empty($rawConfig['test-taker-review-default-open']),
-                'itemTitle' => isset($rawConfig['test-taker-review-item-title']) ? $rawConfig['test-taker-review-item-title'] : null,
-                'preventsUnseen' => !empty($rawConfig['test-taker-review-prevents-unseen']),
-                'canCollapse' => !empty($rawConfig['test-taker-review-can-collapse']),
-                'displaySubsectionTitle' => !empty($rawConfig['test-taker-review-display-subsection-title']),
+                'enabled' => !empty($rawConfig->{'test-taker-review'}),
+                'scope' => isset($rawConfig->{'test-taker-review-scope'}) ? $rawConfig->{'test-taker-review-scope'} : null,
+                'useTitle' => !empty($rawConfig->{'test-taker-review-use-title'}),
+                'forceTitle' => !empty($rawConfig->{'test-taker-review-force-title'}),
+                'showLegend' => !empty($rawConfig->{'test-taker-review-show-legend'}),
+                'defaultOpen' => !empty($rawConfig->{'test-taker-review-default-open'}),
+                'itemTitle' => isset($rawConfig->{'test-taker-review-item-title'}) ? $rawConfig->{'test-taker-review-item-title'} : null,
+                'preventsUnseen' => !empty($rawConfig->{'test-taker-review-prevents-unseen'}),
+                'canCollapse' => !empty($rawConfig->{'test-taker-review-can-collapse'}),
+                'displaySubsectionTitle' => !empty($rawConfig->{'test-taker-review-display-subsection-title'}),
             ],
-            'exitButton' => !empty($rawConfig['exitButton']),
-            'nextSection' => !empty($rawConfig['next-section']),
-            'plugins' => isset($rawConfig['plugins']) ? $rawConfig['plugins'] : null,
+            'exitButton' => !empty($rawConfig->{'exitButton'}),
+            'nextSection' => !empty($rawConfig->{'next-section'}),
+            'plugins' => isset($rawConfig->{'plugins'}) ? $rawConfig->{'plugins'} : null,
             'security' => [
-                'csrfToken' => isset($rawConfig['csrf-token']) ? $rawConfig['csrf-token'] : false,
+                'csrfToken' => isset($rawConfig->{'csrf-token'}) ? $rawConfig->{'csrf-token'} : false,
             ],
             'timer' => [
-                'target' => isset($rawConfig['timer']) && isset($rawConfig['timer']['target']) ? $rawConfig['timer']['target'] : null,
-                'resetAfterResume' => !empty($rawConfig['reset-timer-after-resume']),
-                'keepUpToTimeout' => !empty($rawConfig['keep-timer-up-to-timeout']),
+                'target' => isset($rawConfig->{'timer'}) && isset($rawConfig->{'timer'}['target']) ? $rawConfig->{'timer'}['target'] : null,
```

Comment:
I didn't review whether `$rawConfig`'s properties were objects or arrays. So, perhaps this should be `$rawConfig->{'timer'}->{'target'}`?